### ISSUE TITLE
[BUG] Darts no longer works on python versions lower than 3.8.

### ISSUE THREAD

**Issue by JuanPabloSilvaSoler** [unit8co/darts#1444]
**Describe the bug**
Darts TimeSeries no longer works on python versions lower than 3.8. 

In the new release, typing.Literal was added to the imports in timeseries.py .. But Literal was added to typing.py in 3.8. 

**To Reproduce**
In an environment with python less than 3.8, try from darts import TimeSeries

**System (please complete the following information):**
 - Python version: 3.7.6
 - darts version: 0.23.0

**Additional context**
To use Literal in older versions we could do this:

```
try:
    from typing import Literal
except ImportError:
    from typing_extensions import Literal
```

https://stackoverflow.com/questions/61206437/importerror-cannot-import-name-literal-from-typing

**Comment by dennisbader** [comment]
Hi @JuanPabloSilvaSoler and thanks for raising this.

It will be fixed with #1449.

**Comment by dennisbader** [comment]
Agreed @hrzn. Opened ticket #1456.
